PaymentHighway/Extensions.swift

```diff
@@ -40,17 +39,17 @@ internal extension String {
     /// - returns: Found matches as an array
     func matchesForRegex(_ regex: String!) -> [String] {
```

Comment:
Is the unwrap here needed?

Reply:
Not really! Was there already. i can remove it.